bin/cdklocal

```diff
@@ -91,13 +92,29 @@ getMethods(CdkToolkit.prototype).forEach(meth => {
   }
 });
 
-Object.defineProperty(ToolkitInfo.prototype, 'bucketUrl', {
-  get() {
-    const bucket = this.requireOutput(BUCKET_NAME_OUTPUT);
-    const domain = this.requireOutput(BUCKET_DOMAIN_NAME_OUTPUT) || getLocalHost();
-    return `https://${domain.replace(`${bucket}.`, '')}/${bucket}`
-  }
-});
+function setBucketUrl(object) {
+  Object.defineProperty(object, 'bucketUrl', {
+    get() {
+      const bucket = this.requireOutput(BUCKET_NAME_OUTPUT);
+      const domain = this.requireOutput(BUCKET_DOMAIN_NAME_OUTPUT) || getLocalHost();
+      return `https://${domain.replace(`${bucket}.`, '')}:${port}/${bucket}`
+    }
+  });
+}
+
```

Comment:
Looks awesome, thanks for this PR @AminFazlMondo ! 👍 

Just one remark: To remain compatible with the existing logic, shouldn't we call the following here:
```
setBucketUrl(ToolkitInfo.prototype);
```
... or do you think that's not required?

(As in the previous version we're also calling `Object.defineProperty(ToolkitInfo.prototype ...` at the global scope, at initialization time.)

Reply:
Thanks @whummer for speedy response, make sense in terms of backwards compatibility.
I will do the version required changes soon.